client/daemon/storage/storage_manager.go

```diff
@@ -143,6 +150,9 @@ func NewStorageManager(storeStrategy config.StoreStrategy, opt *config.StorageOp
 		tasks:         &sync.Map{},
 		dataPathStat:  stat.Sys().(*syscall.Stat_t),
 		gcCallback:    gcCallback,
+
+		indexRWMutex:       sync.RWMutex{},
```

Comment:
![image](https://user-images.githubusercontent.com/16535906/122325650-8f6b1580-cf5d-11eb-8294-4e348dd0550a.png)

Reply:
No need to initialize